assets/exchange-rates/latest.json

```diff
@@ -1,180 +1,180 @@
 {
   "disclaimer": "Exchange rates provided for informational purposes only and do not constitute financial advice of any kind. Although every attempt is made to ensure quality, no guarantees are made of accuracy, validity, availability, or fitness for any purpose. All usage subject to acceptance of Terms: https://openexchangerates.org/terms/",
   "license": "Data sourced from various providers; resale prohibited; no warranties given of any kind. All usage subject to License Agreement: https://openexchangerates.org/license/",
-  "timestamp": 1480647611,
+  "timestamp": 1481238011,
```

Comment:
We can not commit it, but I like to commit it from time to time to update the currency rate conversion so we get slightly better metrics.

When I do this, I usually make a separate commit.

This should be harmless though, right? Are you thinking this is an issue?